server/server.js

```diff
@@ -55,12 +55,9 @@ app.set("port", process.env.PORT || 3000);
 app.set("etag", false);
 
 // middlewares
+app.use(redirectSSL);
 app.use(bodyParser.urlencoded({ extended: true }));
-app.use(hsts());
-
-if(process.env.NODE_ENV === "production") {
-  app.use(express_enforces_ssl());
-}
+app.use(helmet());
```

Comment:
I think this needs to be placed before body parser, so that the security headers are still emitted even if body parser fails.